static/app/views/explore/components/table.tsx

```diff
@@ -20,6 +20,7 @@ import {Actions} from 'sentry/views/discover/table/cellAction';
 
 interface TableProps extends React.ComponentProps<typeof _TableWrapper> {
```

Comment:
If you fix the type of `Body` to include the prop, then the `React.ComponentProps<typeof _TableWrapper>` should catch it